server/auth/store.go

```diff
@@ -682,11 +687,8 @@ func (as *authStore) RoleRevokePermission(r *pb.AuthRoleRevokePermissionRequest)
 
 	tx.UnsafePutRole(updatedRole)
 
-	// TODO(mitake): currently single role update invalidates every cache
```

Comment:
I think the 'cost' mentioned by the TODO still exists... now every single role change is rebuilding the cache from the scratch... that actually might be even more costly on the 'synchronous' path of apply...